src/helpers/AllExternalPools.unit.test.tsx

```diff
@@ -0,0 +1,8 @@
+import { allPools, fetchPoolData } from "src/helpers/AllExternalPools";
+
+describe("Test AllExternPools", () => {
+  it("should return immediately if user address is unknown", async () => {
+    const result = await fetchPoolData("");
+    expect(result).toEqual(allPools);
+  });
+});
```

Comment:
need to rewrite this test too

Reply:
after the code change in https://github.com/OlympusDAO/olympus-frontend/pull/1311/commits/45245329e4f71d463f1d91936a0b9c3344121fb9 this test should not be passing, but it is.

It turns out that the test is not working properly, it's throwing a network error & hitting the `catch` in `fetchPoolData`, which is why the test passes.